src/Model/Car.js

```diff
@@ -0,0 +1,19 @@
+export default class Car {
+
+    constructor(name) {
+        this.name = name;
+        this.progress = 0;
+    }
+
+    getName() {
+        return this.name;
+    }
+
+    getProgress() {
+        return this.progress;
+    }
+
+    updateProgress() {
+        this.progress ++;
```

Comment:
위에서 다른분이 comment 달아주신 것 처럼, `+= 1 `을 사용하는 것이 가독성에 좋아서 보편적으로 많이 사용하는 것 같아요!
이부분은 개인 및 팀의 취향 차이라고 생각합니다~!

```suggestion
        this.progress += 1;
```

Reply:
++보다 훨씬 가독성이 있어보이네요
다음 주차부터는 +=1로 작성해야겠어요!